build/README.md

```diff
@@ -6,8 +6,8 @@
   - *Potential change*: Rename this folder to `pipelines` as part of: https://github.com/dotnet/project-system/issues/7915
 
 #### Notable Files
-- [official.yml](build\official.yml): This file is our official build pipeline that produces the signed packages (VSIX) that used for insertion into Visual Studio.
-- [unit-tests.yml](build\unit-tests.yml): This file is our build pipeline used when validating pull requests into the repo from GitHub.
+- [official.yml](ci\official.yml): This file is our official build pipeline that produces the signed packages (VSIX) that used for insertion into Visual Studio.
+- [unit-tests.yml](ci\unit-tests.yml): This file is our build pipeline used when validating pull requests into the repo from GitHub.
```

Comment:
I take it this is fixing a broken link that is now being reported as broken?

Reply:
Correct. This PR both fixed reporting broken links and fixed the links. If you look at the failed commits in this PR, you can see the runs of the GitHub action, reporting links like this one being broken. Click on the little red ❌ from the commits above to see those runs:
![image](https://user-images.githubusercontent.com/17788297/179630664-b4977eab-011a-413e-a488-a46e4ecf1fc3.png)